src/components/theme/ContentMetadataTags/ContentMetadataTags.jsx

```diff
@@ -45,10 +45,28 @@ const ContentMetadataTags = (props) => {
 
   const contentImageInfo = getContentImageInfo();
 
+  const getTitle = () => {
+    const includeSiteTitle =
+      config?.settings?.siteTitleFormat?.includeSiteTitle || false;
+    const titleAndSiteTitleSeparator =
+      config?.settings?.titleAndSiteTitleSeparator || '-';
+
+    const navRootTitle = props.content['@components']?.navroot?.navroot?.title;
+    const siteRootTitle =
+      props.content['@components']?.site?.['plone.site_logo'];
```

Comment:
I tested and it works, but I have one tiny question. Why plone.site_logo here?

Reply:
Sorry, it's a mistake :wink:

Reply:
fixed,